presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizersFactory.java

```diff
@@ -87,6 +88,8 @@ public PlanOptimizersFactory(Metadata metadata, SqlParser sqlParser, FeaturesCon
                 new IndexJoinOptimizer(metadata), // Run this after projections and filters have been fully simplified and pushed down
                 new CountConstantOptimizer(),
                 new WindowFilterPushDown(metadata), // This must run after PredicatePushDown and LimitPushDown so that it squashes any successive filter nodes and limits
+                new MergeIdenticalWindows(),
+                new PruneUnreferencedOutputs(),
```

Comment:
You only need to run PruneUnreferencedOutputs if your optimizer might eliminate symbols that were previously needed. Did you find a case where this was the case?